Automattic/jetpack
# VideoPress: Fix filename with token

Fixes #29390

## Proposed changes:

* Fixes the filename

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the video details page of a private video
* Check that the token is not in the file name

Before | After
-|-
![2023-03-09_12-53-40](https://user-images.githubusercontent.com/8486249/224080019-186bc675-a921-4777-919a-51d0635d7024.png) | ![2023-03-09_12-53-09](https://user-images.githubusercontent.com/8486249/224080053-c4358877-87b2-420e-b4ce-394f6d194690.png)
